i18n/zh-cn/docusaurus-plugin-content-blog/skywalking/local-env-setup.sh

```diff
@@ -44,12 +44,12 @@ if [ "$IS_INSTALLED_CRD" = true ]; then
 
   echo "Step3: Installing Higress "
   helm repo add higress.io https://higress.io/helm-charts
-  helm install higress -n higress-system higress.io/higress --devel --create-namespace --render-subchart-notes  --set global.enableIstioAPI=true --set global.kind=true --set higress-console.o11y.enabled=true  --set higress-controller.domain=console.higress.io --set higress-console.admin.password.value=admin
+  helm install higress -n higress-system higress.io/higress --devel --create-namespace --render-subchart-notes  --set global.enableIstioAPI=true --set global.kind=true --set higress-console.o11y.enabled=true  --set higress-controller.domain=console.higress.io --set higress-console.admin.password=admin
```

Comment:
Is this parameter `higress-console.admin.password`  still needed?

Reply:
If removing it, the following steps need to be updated as well (regarding the login part).